libeos-updater-util/flatpak.h

```diff
@@ -125,6 +125,7 @@ gchar *euu_lookup_flatpak_remote_for_collection_id (FlatpakInstallation  *instal
 
 const gchar *euu_pending_flatpak_deployments_state_path (void);
 const gchar *euu_flatpak_autoinstall_override_paths (void);
+const gchar * euu_get_system_architecture_string (void);
```

Comment:
Nitpick: Unnecessary space after `*`. Please fix in a follow-up PR (or push a fix for this one issue directly).

Reply:
(Done in #155.)